xray/commands/audit/generic/auditmanager.go

```diff
@@ -21,8 +22,54 @@ import (
 	"github.com/jfrog/jfrog-client-go/xray/services"
 )
 
-// GenericAudit audits the project found in the current directory using Xray.
+// GenericAudit audits all the projects found in the given workingDirs
 func GenericAudit(
+	xrayGraphScanParams services.XrayGraphScanParams,
+	serverDetails *config.ServerDetails,
+	excludeTestDeps,
+	useWrapper,
+	insecureTls bool,
+	args []string,
+	progress ioUtils.ProgressMgr,
+	requirementsFile string,
+	ignoreConfigFile bool,
+	workingDirs []string,
+	technologies ...string) (results []services.ScanResponse, isMultipleRoot bool, err error) {
```

Comment:
Let's add a new-line after this line, to make it easier to see where the function's body starts.